### ISSUE TITLE
List Group Item Active makes text not visible

### ISSUE THREAD

**Issue by lordofscripts** [twbs/bootstrap#22590]
I am using the Twitter Bootstrap version 3.3.7 that comes in the Visual Studio 2017 website template, I have not modified it nor override any bootstrap class.

I was playing with anchored List Group Items within Panels and noticed that when I use just _**list-group-item**_ in a LI element it displays fine. However, if I use **_list-group-item active_** to mark it as the currently selected item,  Then the item is highlighted (active) but the foreground color appears to be the same as the background color because the text is NOT visible.  It does not matter if it is a default panel or a primary panel.

Basically this works fine: 
             `<li class="list-group-item"> <i class="fa fa-ship"></i> <a ....>TEXT</a>  </li>`
But this makes the foreground TEXT not visible:
             `<li class="list-group-item active"> <i class="fa fa-ship"></i> <a ....>TEXT</a>  </li>`

The reason why I do not use list-group on a DIV but use a UL element instead is that with the DIV approach every line (Font Awesome icon plus anchor) is split in two, icon above, anchor below but when put in a LI they are kept together on the same line.

**Comment by lordofscripts** [comment]
Do note however that if I use list-group in a DIV rather than an UL element it works fine. However, It seems more natural to have LISTS with ANCHORS in Panels (menus) so my use case is not really abnormal.

**Comment by Johann-S** [comment]
Bug reports must include a live demo of the problem. Per our contributing guidelines, please create a reduced test case via [JS Bin](https://jsbin.com/) or [CodePen](http://codepen.io/) and report back with your link, Bootstrap version, and specific browser and OS details.

**Comment by Johann-S** [comment]
Bootstrap 3 is essentially in maintenance mode as we focus on working towards a stable v4. As such, we're only accepting changes to v3's code on a case-by-case basis, usually only for critical bug fixes or docs improvements.

**Comment by lordofscripts** [comment]
DISREGARD. Found a way to get it done that way. Sorry!.
